src/modules/Elsa.Workflows.Management/Activities/WorkflowDefinitionActivity/WorkflowDefinitionActivity.cs

```diff
@@ -165,10 +165,10 @@ await workflowDefinitionService.FindWorkflowGraphAsync(filter, cancellationToken
         return workflowGraph;
     }
 
-    private ActivityDescriptor FindActivityDescriptor(IServiceProvider serviceProvider)
+    private async Task<ActivityDescriptor> FindActivityDescriptor(IServiceProvider serviceProvider)
```

Comment:
Please mind the naming convention to apply the `Async` suffix when changing methods into async.